core/core_test.go

```diff
@@ -33,7 +33,9 @@ const (
 )
 
 func TestEthereumChainDetection(t *testing.T) {
-	meshDB, err := meshdb.New("/tmp/meshdb_testing/" + uuid.New().String())
+	contractAddresses, err := ethereum.NewContractAddressesForChainID(constants.TestChainID)
```

Comment:
I see this pattern a lot in the tests:

```
	contractAddresses, err := ethereum.NewContractAddressesForChainID(constants.TestChainID)
	require.NoError(t, err)
```

We don't really need to go through the hoops of looking up the contract addresses for `constants.TestChainID` since we know that the addresses are defined. Can you make a helper method in __ethereum/contract_addresses__ that just returns the addresses for Ganache and doesn't return an error?

Reply:
Good catch!